xcodeproj/internal/default_automatic_target_processing_aspect.bzl

```diff
@@ -174,6 +177,7 @@ def _default_automatic_target_processing_aspect_impl(target, ctx):
             launchdplists = launchdplists,
             entitlements = entitlements,
             bazel_build_mode_error = bazel_build_mode_error,
+            app_icons = app_icons,
```

Comment:
We should have done it earlier, but can you alphabetize this whole constructor call?